### ISSUE TITLE
ERROR: MIDI Compiler: (EVENTS):

### ISSUE THREAD

**Issue by kerusa656** [mtolly/onyxite-customs#151]
Hi @mtolly  im testing more... and this error hang on....

# Running Magma v2 to export MIDI
ERROR! process exited with code 1
stdout:
Reading the archive
SystemInit Params: C:\Users\AppData\Local\Temp\magma-bcff8d0fe87c81d9\MagmaCompilerC3.exe -export_midi C:\Users\AppData\Local\Temp\onyx-1a7bf2556410d9f8\gen/target\87822088\magma/magma.rbproj C:/Users/AppData/Local/Temp/onyx-1a7bf2556410d9f8/gen/target/87822088/notes-magma-export.mid
MIDI Compiler: Starting...
ERROR: MIDI Compiler: (EVENTS): MIDI events at [70:4:015] after the end event at [70:4:000]
MIDI Compiler: Done.

stderr:

Context (innermost first):
  - running Magma v2 to export MIDI
  - shake: C:/Users/AppData/Local/Temp/onyx-1a7bf2556410d9f8/gen/target/87822088/notes-magma-export.mid
  - shake: C:/Users/AppData/Local/Temp/onyx-1a7bf2556410d9f8/gen/target/87822088/notes-magma-added.mid
  - shake: C:/Users/AppData/Local/Temp/onyx-1a7bf2556410d9f8/gen/target/87822088/stfs/songs/o794370671/o794370671.mid
  - shake: C:/Users/AppData/Local/Temp/onyx-1a7bf2556410d9f8/gen/target/87822088/rb3con

**Comment by sonicbro** [comment]
That an error with your chart. 

**ERROR: MIDI Compiler: (EVENTS): MIDI events at [70:4:015] after the end event at [70:4:000]**
Says it right there.

**Comment by kerusa656** [comment]
@sonicbro most of these errors happend in oficial CON DLC files...

**Comment by sonicbro** [comment]
Right, it just means that the midi was reinserted after the con/project file had been created. Since Magma would've returned an error value similar to yours.

I don't think the game actually cares about a stray event being called for after the [end] event, since by that point in the song, the game ends. 

Drag the CON file into C3 tools, dig into the files until you find the mid file. Check the midi file's EVENT track. See what the issue is and correct it.
